travel/res_config.py

```diff
@@ -36,6 +38,26 @@ def get_basic_passenger_limit(ir_config_parameter_pool, cr, uid, context=None):
         return limit
 
 
+def get_alert_address(ir_config_parameter_pool, cr, uid, context=None):
+    context = context or {}
```

Comment:
With `{}`. The reference would be different, which means if something were later added to context, those changes wouldn't go back up the stack.

```python
In [1]: context = {}

In [2]: id(context)
Out[2]: 140239108613576

In [3]: context = context or {}

In [4]: id(context)
Out[4]: 140239152358016

```

Is this what you mean?